Clinical-Genomics/scout
# Added GENS integration

This PR integrates Gens to scout. When the `GENS_HOST` variable i set in the config links to Gens are displayed in case and structural variant views.

You could optionally verify the integrity of the links by installing Gens v1.1.1 to a local vm. I would recommend using the docker image.

**How to test**:
1. goto cust000/643594, verify that "View CN profile" button is absent
2. goto cust000/643594/sv/variants/77d69d4d78a8e272365bdabe4f607327
3. verify that copy number profile section is absent
4. set GENS_HOST and/or GENS_PORT variables in configuration
5. goto previous pages and verify that button is present

**Expected outcome**:
The button is only present when the HOST is configured.

**Review:**
